Cargo.toml

```diff
@@ -33,4 +33,6 @@ cupid = "0.4.0"
 [features]
 strict = []
 std = []
+# This feature enables only those intrinsics supported by
+# Intel's Software Development Environment (SDE):
```

Comment:
Thanks! Could this extend the documentation to indicate that this is an internal-only feature not intended for consumers?

Additionally is the `strict` feature like that as well, where it's' internal-only? If so mind noting that as well in addition to adding some docs to `strict`? (this can happen in a future PR though)

Reply:
Yes both `intel_sde` and `strict` are internal features, is there a way to add development-only features? 

Or is there a standard convention for documenting these? 

I would otherwise just add `# internal only: explanation`.

Reply:
Right now the only way of having a development-only feature is to specify it through `RUSTFLAGS` but that's not really the best solution. A comment I think should work just fine!

Reply:
done :)